src/fullscreen/fullscreen-launch.directive.ts

```diff
@@ -0,0 +1,4 @@
+import { Directive } from '@angular/core';
+
+@Directive({ selector: '[fullscreen-launch]' })
```

Comment:
I suggest change selector to `thyFullscreenLaunch, [fullscreen-launch]`,  use CamelCase name all directive selector and add prefix `thy`, for backward compatibility, add `[fullscreen-launch]` temporarily, remove it in later version, and please update examples use `thyFullscreenLaunch`, Thank you!